src/pages/AdminTemplates/AdminTemplates.scss

```diff
@@ -0,0 +1,169 @@
+@import "scss/constants.scss";
+
+.templates-page {
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 5rem;
+
+  &__nav-buttons {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 80%;
+
+    &--cancel {
+      padding: 0.5rem 1.5rem;
+      border-radius: 22px;
+      background-color: lighten($color: $saturated-black, $amount: 10);
+      color: $white;
+      border: none;
+
+      &:hover {
+        background-color: lighten($color: $saturated-black, $amount: 20);
+      }
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    &--confirm {
+      padding: 0.5rem 1.5rem;
+      border-radius: 22px;
+      background-color: $primary;
+      color: $white;
+      border: none;
+
+      &:hover {
+        background-color: lighten($color: $primary, $amount: 10);
+      }
+
+      &:focus {
+        outline: none;
+      }
+    }
+  }
+
+  &__template-preview {
+    width: 60%;
+    height: 60vh;
+    border-radius: 22px;
+    padding: 5rem;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__title {
+    font-size: 2rem;
+    font-weight: bold;
+    padding: 1rem 0 0.5rem 0;
+  }
+
+  &__subtitle {
+    font-size: 1rem;
+    opacity: 0.7;
+    padding-bottom: 1rem;
+  }
+
+  &__header-button {
+    padding: 0.5rem 1.5rem;
+    border-radius: 22px;
+    background-color: lighten($color: $saturated-black, $amount: 10);
+    color: $white;
+    border: none;
+
+    &:hover {
+      background-color: lighten($color: $saturated-black, $amount: 20);
+    }
+
+    &--disabled {
+      opacity: 0.5;
+    }
+  }
+
+  &__recipes {
+    border-radius: 22px;
+    background-color: lighten($color: $saturated-black, $amount: 10);
+    color: $white;
+    border: none;
+  }
+
+  &__recipe {
```

Comment:
I would have to look closer at the structure of the code, but there seem to be a lot of `__elements` here that i'm guessing may make sense to be extracted into sub-components, rather than all combined into this one file.